fe1-web/src/core/network/__tests__/JsonRpcApi.test.ts

```diff
@@ -151,13 +152,13 @@ describe('catchup', () => {
 
     const expected = ExtendedMessage.fromMessage(mockResponseMessage, mockChannel, mockAddress);
 
-    // the receivedAt value can differ
-    // @ts-ignore
-    delete value.receivedAt;
-    // @ts-ignore
-    delete expected.receivedAt;
-
-    expect(value).toBeJsonEqual(expected);
+    // the receivedAt value is allowed differ
+    // cannot use expect.any(Number) here because of toBeJsonEqual and cannot use toBeEqual
+    // because once it is an instance of Base64Url and once of Hash
+    expect({ ...value, receivedAt: 0 }).toBeJsonEqual({
+      ...expected,
+      receivedAt: 0,
+    });
```

Comment:
```suggestion
    // the receivedAt value is allowed differ
    expect({ ...value, receivedAt: 0 })
      .toBeJsonEqual({...expected, receivedAt: 0 });
```

I don't think the comments about expect.any / toBeJsonEqual / etc. will make much sense to someone who's not actively fighting with jest on these few lines of code 😅 but the comment itself is very useful to a reviewer.
In general, I try to put these comments as a self-review in the PR itself, so that the reviewers can benefit from the context, but it's also not confusing a subsequent developer reading through this code.

Reply:
That makes sense!